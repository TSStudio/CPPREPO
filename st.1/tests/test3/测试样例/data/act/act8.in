5
tony 8
roy 9
resy 7
hanke 5
miae 3