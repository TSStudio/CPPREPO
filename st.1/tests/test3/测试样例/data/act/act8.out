miae
hanke
resy
tony
roy